tests/test_application/test_application.py

```diff
@@ -77,6 +77,21 @@ def test_application_contents_with_none_contents(flask_test_client):
 
     assert response.status_code == 400
 
+@pytest.mark.usefixtures("live_server")
+def test_application_upload_file_response(flask_test_client):
+    """
+    GIVEN: our service running on flask test client.
+    WHEN: we post expected data with type "file" to endpoint "/send".
+    THEN: we check if it returns status code 200.
+    """
+
+    response = flask_test_client.post(
+        "/send",
+        json=expected_application_data,
```

Comment:
Can we actually assert on the contents of the answer for the file key rather than it just returning a 200?

Reply:
we cant check contents because all the contents are in the attached file link so checking 200 response

Reply:
Can we unit test a function to test the output rather than this integration test?